src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -771,7 +771,8 @@ protected override void DecodeWellKnownAttributeImpl(ref DecodeWellKnownAttribut
                 ReservedAttributes.IsByRefLikeAttribute |
                 ReservedAttributes.TupleElementNamesAttribute |
                 ReservedAttributes.NullableAttribute |
-                ReservedAttributes.NativeIntegerAttribute))
```

Comment:
What about on method/return?

Reply:
`[ScopedRef]` is not expected on methods or returns. I'll add a test for `[return: ScopedRef]` since it looks like that case is untested.

Reply:
Added `[return: ScopedRef]` test to `AttributeTests_ScopedRef.ExplicitAttribute_ReferencedInSource_02()` in [https://github.com/dotnet/roslyn/pull/62886](https://github.com/dotnet/roslyn/pull/62886).